python/astro_metadata_translator/properties.py

```diff
@@ -336,4 +336,25 @@ class PropertyDefinition:
         "int",
         int,
     ),
+    "has_simulated_content": PropertyDefinition(
+        "Boolean indicating whether any part of this observation was" " simulated.", "bool", bool, None, None
+    ),
+    "group_counter_start": PropertyDefinition(
+        "Observation counter for the start of the exposure group."
+        "Depending on the instrument the relevant group may be "
+        "visit_id or exposure_group.",
+        "int",
+        int,
+        None,
+        None,
+    ),
+    "group_counter_end": PropertyDefinition(
```

Comment:
A modern visit is defined by the tuple of start_num, end_num. If the second observation does not turn up as expected the next observation will have a different start_num (or will not be there at all) so there won't be a problem. When visits are defined it will define the visit with just the first one in. I do complain in define-visits if I get the second file of a pair since that implies the first one must have been taken.